src/Native/Windows/clrcompression/CMakeLists.txt

```diff
@@ -74,3 +74,7 @@ GENERATE_EXPORT_HEADER( clrcompression
 )
 
 install (TARGETS clrcompression DESTINATION .)
+
+install (FILES ${CMAKE_CURRENT_BINARY_DIR}/Debug/clrcompression.pdb DESTINATION .
+    CONFIGURATIONS Debug
```

Comment:
When I do `build native /p:ConfigurationGroup=Release`, I only get a `dll`, `exp`, and `lib` in `bin\obj`. This install doesn't run, so there's only the `dll` and `lib` in `bin`. When I build the packages, I get the normal library package with the `dll`, but since there's no `pdb` NuGetPack skips making the symbol package.

Do you think that we should have a `pdb` for release builds as well? I'm not familiar enough with native builds to know when we want one and when we can get one. (It would be simple to remove `CONFIGURATIONS Debug` and let the pdb flow through if someone can help me get the pdb created.)